library/src/main/java/io/github/douglasjunior/androidSimpleTooltip/SimpleTooltip.java

```diff
@@ -269,7 +279,11 @@ private void configContentView() {
                 tv.setText(mText);
         }
 
-        mContentView.setPadding((int) mPadding, (int) mPadding, (int) mPadding, (int) mPadding);
+        if (mPadding != mDefaultPadding) {
```

Comment:
Take the `padding` variable if it is set. Or just use the left, top, right and bottom values. Their default value is again mDefaultPadding.

Reply:
Sorry for the delay.

I thought of something much simpler.

Just add the method in the builder:

```java
         public Builder padding(float padding) {
             this.paddingTop = padding;
             this.paddingBottom = padding;
             this.paddingLeft = padding;
             this.paddingRight = padding;
             return this;
         }
```

It's just to maintain compatibility with anyone who already uses the `padding()` method. And you can eliminates the `padding` variable.